receiver/prometheusremotewritereceiver/receiver.go

```diff
@@ -122,3 +153,73 @@ func (prw *prometheusRemoteWriteReceiver) parseProto(contentType string) (promco
 	// No "proto=" parameter found, assume v1.
 	return promconfig.RemoteWriteProtoMsgV1, nil
 }
+
+// translateV2 translates a v2 remote-write request into OTLP metrics.
+// translate is not feature complete.
+// nolint
+func (prw *prometheusRemoteWriteReceiver) translateV2(_ context.Context, req *writev2.Request) (pmetric.Metrics, promremote.WriteResponseStats, error) {
+	var (
+		badRequestErrors []error
+		otelMetrics      = pmetric.NewMetrics()
+		b                = labels.NewScratchBuilder(0)
+		stats            = promremote.WriteResponseStats{}
+	)
+
+
+	for _, ts := range req.Timeseries {
+		ls := ts.ToLabels(&b, req.Symbols)
+
+		if !ls.Has(labels.MetricName) {
+			badRequestErrors = append(badRequestErrors, fmt.Errorf("missing metric name in labels"))
+			continue
+		} else if duplicateLabel, hasDuplicate := ls.HasDuplicateLabelNames(); hasDuplicate {
+			badRequestErrors = append(badRequestErrors, fmt.Errorf("duplicate label %q in labels", duplicateLabel))
+			continue
+		}
+
+		var rm pmetric.ResourceMetrics
+		// This cache should be populated by the metric 'target_info', but we're not handling it yet.
+		cacheEntry, ok := prw.jobInstanceCache[ls.Get("instance")+ls.Get("job")]
+		if ok {
+			rm = pmetric.NewResourceMetrics()
+			cacheEntry.CopyTo(rm)
+		} else {
+			// A remote-write request can have multiple timeseries with the same instance and job labels.
+			// While they are different timeseries in Prometheus, we're handling it as the same OTLP metric
+			// until we support 'target_info'.
+			rm = otelMetrics.ResourceMetrics().AppendEmpty()
+			parseJobAndInstance(rm.Resource().Attributes(), ls.Get("instance"), ls.Get("job"))
+			prw.jobInstanceCache[ls.Get("instance")+ls.Get("job")] = rm
+
+
+			scopeAttributes := rm.ScopeMetrics().AppendEmpty().Scope().Attributes()
+			for _, l := range ls {
+				if l.Name == "instance" || l.Name == "job" || l.Name == labels.MetricName {
```

Comment:
We should also omit `otel_scope_name` and `otel_scope_version`, which should be used to set the name and version of the scope (if present).